### DISCUSSION TITLE
Map/Document used versions tractusx-edc/connector-edc/management-api

### DISCUSSION THREAD

**Discussion by stephanbcbauer** [eclipse-tractusx/tractusx-edc#186] [General]
**current situation**

Right now, we don't have a “documented” mapping between tractusx-edc/connector-edc/managament-api versions. Best example is the management-api which was initially used for documentation https://eclipse-tractusx.github.io/docs/kits/product-edc/docs/kit/development-view/openAPI/management-api/management-api

I created the MDX files out of an openAPI.yaml, which is located under upstream EDC. I am just wondering how do we get informed about new versions and changes. And how can we automate the documentation/generation.

How can we document the used versions tractusx-edc/connector-edc/managament-api

**Comment by ndr-brt** [comment]
Currently we are using EDC nightly versions in tractusx, and unfortunately they don't have a tag that could be used to get the correct `openapi.yaml`, for the moment, as a fix, we could use the commit hash of the EDC version that was used to get the `openapi.yaml` file

**Reply by stephanbcbauer** [reply]
@ndr-brt @paullatzelsperger will schedule a meeting for us, just to provide a result here :)

**Comment by stephanbcbauer** [comment]
Mapping is given, the managament-api (openAPI.yaml) has a version included, for example für tractusx-edc version 0.5.x the API version 0.1.3 is used -> https://eclipse-tractusx.github.io/docs-kits/next/kits/tractusx-edc/docs/kit/development-view/openAPI/control-api/control-api

Discussion can be closed. Thx

**Reply by arnoweiss** [reply]
The Links to the kit are outdated. Can you update them?

**Reply by stephanbcbauer** [reply]
[Mapping is given, the managament-api (openAPI.yaml) has a version included, for example für tractusx-edc version 0.5.x the API version [0.5.1](http://localhost:3000/docs-kits/next/kits/tractusx-edc/docs/kit/development-view/openAPI/tractusx-edc-api/tractus-x-edc-rest-api)

**Reply by stephanbcbauer** [reply]
@arnoweiss since the OpenAPI plug-in of docusaurus has problems… we will probably switch/link to [SwaggerHub](https://app.swaggerhub.com/apis/eclipse-tractusx-bot/tractusx-edc/0.5.1-SNAPSHOT)
